src/Pact/Native/Keysets.hs

```diff
@@ -49,6 +52,10 @@ keyDefs =
      "Define keyset as NAME with KEYSET, or if unspecified, read NAME from message payload as keyset, \
      \similarly to 'read-keyset'. \
      \If keyset NAME already exists, keyset will be enforced before updating to new value."
+    ,setTopLevelOnly $ defGasRNative "describe-keyset" descKeySet
+     (funType tTyObjectAny [("keyset",tTyString)]) [] "Get metadata for KEYSET."
+    ,setTopLevelOnly $ defGasRNative "describe-guard" descGuard
+     (funType tTyObjectAny [("keyset",tTyString)]) [] "Get metadata for KEYSET."
```

Comment:
```suggestion
     (funType tTyObjectAny [("guard",tTyString)]) 
     ["(env-data {'k: ['abc]}) (describe-guard (read-keyset 'k))"]
     "Get metadata for GUARD."
```

You can run Pact.DocGen to test your examples, or just run the full test suite. In general when working on builtins this is necessary (running full testsuite) to generate docs for the new builtin.

Reply:
We can build out these docs more once we agree on the spec